### ISSUE TITLE
Kotlin plugin version not compatible

### ISSUE THREAD

**Issue by Abduraimbek** [pr0gramista/charset_converter#20]
Following error occurs while bulding android project:

___
FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':charset_converter' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50
___

My kotlin version is **1.7.20**
Gradle version is **7.3.1**

```
buildscript {
    ext.kotlin_version = '1.7.20'
    repositories {
        google()
        mavenCentral()
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:7.3.1'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}
```

**Comment by pr0gramista** [comment]
Sorry for the late response. Have you found a solution? I'd be happy to fix it so others don't have to.

I'm not sure what's the recommended Kotlin version for the plugins, but I'll check some defaults the Flutter CLI has right now.

**Comment by Abduraimbek** [comment]
Hi Bartosz Wiśniewski! Thanks for reply. I solved problem by downgrading my kotlin version. But it uses some deprecated APIs.

**Comment by pr0gramista** [comment]
I just released version 2.1.1 with upgraded Kotlin and Gradle versions which I believe should resolve this issue. Let me know if it doesn't fix it.

Special thanks to the people who contributed #22 and #23 💙
